### PR TITLE
Add a test for FCC conditions for none FPU instructions.

### PR DESCRIPTION
**Pull request by Rot127**
**Your checklist for this pull request**
- [ ] I've documented or updated the documentation of every API function and struct this PR changes.
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)

**Detailed description**

Just a little paranoid because the CC fields are a little complex to handle.
So here an extra test.

**Test plan**

...

**Closing issues**

...

### CODE REVIEW

File: tests/details/sparc.yaml [hunk 1]
```diff
@@ -927,3 +927,26 @@ test_cases:
...
+      arch: "sparc"
+      options: [ CS_OPT_DETAIL, CS_MODE_BIG_ENDIAN, CS_MODE_V9 ]
+      address: 0x0
+    expected:
+      insns:
+        -
+          asm_text: "movu	%fcc0, %l1, %l2"
+          details:
+            sparc:
+              operands:
```
Comment (jiegec) [inline tests/details/sparc.yaml:942]: It is a conditional move instruction that reads fcc0 and conditionally copies l1 to l2. Why is fcc0 not added as the first operand?

Comment (Rot127) [reply tests/details/sparc.yaml:942]: It is the CC field which is checked. This is the `cc_field` below.

Comment (Rot127) [reply tests/details/sparc.yaml:942]: The cc fields are itself not an operand in our classical interpretation. But is a field in some register. For `fcc0`, `fcc1` tc it is the `fsr` register. for `xcc` and `icc` it is the `ccr` register.

There is a conflict now: is the operand the `ccr` register or the specific `cc`, the `fcc0` field or `fsr`? It is actually both. But our common interpretation of register operands always refers to the "full-register". I felt like it might be confusing.
Because the `cc` field is commonly encoded as extra detail member as well (not as operand).

So I tried to distinguish this subtle interpretation difference and added the `cc_field` member to the Sparc details instead of treating it as a full operand.
This way the `cc` field is accompanied by the `cc_field`.

**Review by jiegec — APPROVED** [review]
